engine/src/main/java/org/hibernate/validator/internal/engine/ConfigurationImpl.java

```diff
@@ -132,6 +133,16 @@ private ConfigurationImpl() {
 		this.addConstraintDefinitionContributor( defaultConstraintDefinitionContributor );
 	}
 
+	private ValidatedValueUnwrapper<?> createJavaFXUnwrapperClass(TypeResolutionHelper typeResolutionHelper) {
+		try {
+			Class<?> jfxUnwrapperClass = run( LoadClass.action( JFX_UNWRAPPER_CLASS, getClass().getClassLoader() ) );
+			return (ValidatedValueUnwrapper<?>) ( jfxUnwrapperClass.getConstructor( TypeResolutionHelper.class ).newInstance( typeResolutionHelper ) );
```

Comment:
I'm thinking the same, Let's ue the original fix and hear back from the person with the issue, if he still has problem we will reopen the issue.

I'll send another pull request